src/devices/bus/ieee488/ieee488.cpp

```diff
@@ -410,3 +410,14 @@ SLOT_INTERFACE_END
 SLOT_INTERFACE_START(hp_ieee488_devices)
 	SLOT_INTERFACE("hp9895", HP9895)
 SLOT_INTERFACE_END
+
+//-------------------------------------------------
+//  SLOT_INTERFACE( remote488_devices )
+//-------------------------------------------------
+
+// slot devices
+#include "remote488.h"
+
+SLOT_INTERFACE_START(remote488_devices)
+        SLOT_INTERFACE("remote488", REMOTE488)
+SLOT_INTERFACE_END
```

Comment:
Why put this in a separate group?  Doesn't it make sense to live in the groups with other devices for machines it's compatible with?  Does it not work with other kinds of machines with 488 interfaces in MAME?

Reply:
See my comment below